test/dotnet/input/Datatypes/TestNumeric.txt

```diff
@@ -176,3 +176,6 @@ prepst#!#exec#!#numeric|-|a7|-|-123|-|9|-|2
 #SELECT * FROM numeric_table;
 
 DROP TABLE numeric_table;
+
+# Test BABEL-3344
+SELECT CASE WHEN 1=1 THEN 0 ELSE cast(98.7627 AS numeric(24,6)) END
```

Comment:
why we have written test this way where else will never execute

Reply:
This is the original test scenario given in the jira but although the `else` never gets returned but it's value decides the return type of result. So, `0` will be returned as numeric and int.